circuit_knitting/utils/observable_grouping.py

```diff
@@ -45,14 +45,16 @@ def observables_restricted_to_subsystem(
     A :class:`~qiskit.quantum_info.PauliList` will be returned if a :class:`~qiskit.quantum_info.PauliList` is provided; otherwise,
     a ``list[Pauli]`` will be returned.
 
+    Any phase information will be discarded, consistent with the standard behavior when slicing a Pauli.
```

Comment:
You could mention the other function(s) that has this behavior, since a user may not immediately know what all this "slicing a Pauli" means exactly.

Reply:
Here's an example of what I mean that the phase gets discarded:
```python
>>> from qiskit.quantum_info import Pauli
>>> Pauli("iXZX")[:2]
Pauli('ZX')
```
It's the act of slicing, not really a function, but I will try to think of a better explanation that doesn't require somebody to know Python or programming jargon.

Reply:
Ohhh I see. Actual Python slicing. No, this is fine. Thanks